.azure/scale.csv

```diff
@@ -0,0 +1,26 @@
+stage,siteId,city,mobile,country,aksArc-controlPlaneIp,phone,streetAddress1,startingAddress,rdmaEnabled,storageConnectivitySwitchless,aksArc-lnet-startingAddress,rbacAdminGroupObjectIds,serverPorts,aksArc-lnet-addressPrefix,storageNetworks,aksArc-lnet-endingAddress,defaultGateway,managementAdapters,virtualHostIp,enableProvisioners,adouSuffix,companyName,dnsServers,postalCode,stateOrProvince,authenticationMethod,endingAddress,emailList,location,aksArc-lnet-dnsServers,aksArc-lnet-defaultGateway,contactName,dcPort,servers,domainServerIP,domainFqdn
```

Comment:
## Arc IaC Automation generated
Next steps: 
  - [ ] Setup local git hook by running command in the root folder of cloned repo: git config --local core.hooksPath ./.azure/hooks/  
  - [ ] Fill the csv file and commit, the struct of the csv file can be found on [here](https://github.com/Azure/Edge-infrastructure-quickstart-template/blob/main/doc/Scale-with-automation.md#methodology)
  - [ ] Wait for the scale workflow run, and then resolve this comment and merge the pull request

 If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com